pipelines/streaming/README.md

```diff
@@ -0,0 +1,130 @@
+The
+[streaming](https://github.com/google/fhir-data-pipes/tree/master/pipelines/streaming)
+directory contains code to continuously listen for changes to an underlying
+OpenMRS MySQL database using
```

Comment:
You can drop explicit reference to "MySQL" (because I think that is what Debezium is supposed to hide).